src/script.js

```diff
@@ -36,13 +36,27 @@ class Item {
     removeButton.classList.add('removeButton');
     itemBox.appendChild(removeButton);
 
+    const updateButton = document.createElement('button');
+    updateButton.textContent = 'Update';
+    updateButton.classList.add('updateButton');
+    itemBox.appendChild(updateButton);
+
     editButton.addEventListener('click', () => this.edit(input));
     removeButton.addEventListener('click', () => this.remove(itemBox));
+    updateButton.addEventListener('click', () => this.update(input, itemName));
+  }
+
+  update(input, itemName) {
+    this.itemName = input.value;
+    itemList.splice(itemList.indexOf(itemName), 1, this.itemName);
+    this.saveItems();
+    input.disabled = true;
   }
 
   // eslint-disable-next-line class-methods-use-this
   edit(input) {
-    input.disabled = !input.disabled;
+    input.disabled = false;
+    input.focus();
   }
 
```

Comment:
Good work setting the start value of the key completed as false 💯 

- Kindly as the previous reviewer requested, when you add a new task it should create a new item, it does now, but not correctly at the moment

![image](https://user-images.githubusercontent.com/93630700/230620325-06432013-9261-4d02-9377-bd3fe2c2d78d.png)

- It shouldn't show any object on the screen and the items should be added with the properties